src/Avalonia.Base/Input/PointerEventArgs.cs

```diff
@@ -70,6 +70,13 @@ private Point GetPosition(Point pt, Visual? relativeTo)
             if (relativeTo == null)
                 return pt;
 
+            if (!ReferenceEquals(_rootVisual, relativeTo.VisualRoot) && relativeTo.VisualRoot is Visual v)
```

Comment:
Could we get a comment here to explain what it's doing/why it's needed?

Reply:
added comments.